new_ui/app.py

```
@@ -147,6 +147,190 @@ async def get_styles():
+    return {"queued": True, "task_id": id(task)}
+
+
+@app.post("/api/generate/stop")
+async def generate_stop():
+    """Stop the current generation.
+
+    The worker pops the active task from async_tasks before processing,
+    so we must also check the module-level current_task reference.
+    Mirrors the stop logic used in the Gradio UI (webui.py stop_clicked).
+    """
+    import ldm_patched.modules.model_management as model_management
```

Ruff (0.15.9)

[warning] Use `from ldm_patched.modules import model_management` in lieu of alias

Replace with `from ldm_patched.modules import model_management`

(PLR0402)